src/components/admin/CommandExecutionPanel.tsx

```
@@ -0,0 +1,416 @@
+      <div className="border border-gray-200 dark:border-gray-700 rounded-lg bg-white dark:bg-gray-800">
+        <div className="p-4 border-b border-gray-200 dark:border-gray-700">
+          <h3 className="text-lg font-semibold flex items-center gap-2">
+            <Terminal className="h-5 w-5" />
+            Command Execution
+          </h3>
+        </div>
+        <div className="p-4 space-y-4">
+          {/* Command Selection */}
+          <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+            <div>
+              <label className="block text-sm font-medium mb-2">Command</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)